README.md

```diff
@@ -40,11 +41,13 @@ If the `prop` is an object, the default display is the following property of the
 Only the property identified through `prop` is updated, the rest of the model remains
 uncommited.
 
+When an update is successful, the `onUpdate` function is called with the updated model.
+
 #### Keyboard Support
 
 Currently, the only way to confirm an edit is to hit `enter`.
```

Comment:
Do you think a more terse comment would be better here? for example: "Hitting `enter` will confirm an edit"